google/cloud/aiplatform/models.py

```diff
@@ -687,10 +687,10 @@ def _validate_deploy_args(
                 raise ValueError(
                     "Sum of all traffic within traffic split needs to be 100."
                 )
-
-        if bool(explanation_metadata) != bool(explanation_parameters):
+        
+        if bool(explanation_metadata) == True and bool(explanation_parameters) == False:
```

Comment:
nit: it is better to use implicit if true/false if possible. see go/pystyle#truefalse-evaluations 

same for instances below.